web/test.sh

```diff
@@ -21,72 +21,53 @@ builder_describe "Runs the Keyman Engine for Web unit-testing suites" \
   "test+" \
   ":dom                  Runs DOM-oriented unit tests (reduced footprint, nothing browser-specific)" \
   ":integrated           Runs KMW's integration test suite" \
-  "--ci                  Set to utilize CI-based test configurations & reporting.  May not be set with $(builder_term --debug)." \
-  "--reporters=REPORTERS Set to override the 'reporters' used by the unit testing engines" \
-  "--browsers=BROWSERS   Set to override automatic browser selection for $(builder_term :engine) tests"
+  "--ci                  Set to utilize CI-based test configurations & reporting.  May not be set with $(builder_term --debug)."
 
 builder_parse "$@"
 
 # Browser-based tests: common configs & kill-switches
 
-DO_BROWSER_TEST_SUITE=true
-
-if [[ $VERSION_ENVIRONMENT == test ]]; then
-  # Implied: CONFIG=CI.conf.js because `-CI` parameter is passed.
-  #
-  # If we are running a TeamCity test build, for now, only run BrowserStack
-  # tests when on a PR branch with a title including "(web)" or with the label
-  # test-browserstack. This is because the BrowserStack tests are currently
-  # unreliable, and the false positive failures are masking actual failures.
-  #
-  # We do not run BrowserStack tests on master, beta, or stable-x.y test
-  # builds.
-  DO_BROWSER_TEST_SUITE=false
-  if builder_pull_get_details; then
-    if [[ $builder_pull_title =~ \(web\) ]] || builder_pull_has_label test-browserstack; then
-      DO_BROWSER_TEST_SUITE=true
-    fi
-  fi
-fi
-
-function get_default_browser_set() {
-  if [[ $BUILDER_OS == mac ]]; then
-    BROWSERS="--browsers Firefox,Chrome,Safari"
-  elif [[ $BUILDER_OS == win ]]; then
-    BROWSERS="--browsers Firefox,Chrome"
-  else
-    BROWSERS="--browsers Firefox,Chrome"
-  fi
-}
```

Comment:
Source of the current build failure:

./test.sh: line 80: get_default_browser_set: command not found

Looks like I rebased this deletion a bit too early, so the CI check falls over here.  It's fine as of the next PR in the chain though - #11456 passed its checks.